src/components/react-json-schemaForm/GeometryField/Informations.js

```diff
@@ -13,26 +15,34 @@ const getIconFromType = type => {
   return 'polygon';
 };
 
-const Information = props => {
+const Information = ({
+  schema: { default: {
+    coordinates: schemaCoordinates,
+    type: schemaType,
+  } },
+}) => {
   const {
-    formData: { geom } = {},
     isRequired,
     isRouting,
-    schema: { default: {
-      coordinates: schemaCoordinates,
-      type: schemaType,
-    } },
-  } = props;
+    nextFormData: { geom } = {},
+  } = useContext(GeometryFieldContext);
```

Comment:
You can add a `useGeometryField` hook and use it instead. It's a little more clean (only a minor detail)

Reply:
Yes, totally right!
Done 👍